src/main/scala/com/ubirch/webui/models/authentification/BearerAuthStrategy.scala

```diff
@@ -162,7 +163,7 @@ trait AuthenticationSupport extends ScentrySupport[(UserInfo, MemberType)] with
     (for {
       claims <- authSystems()
       user <- Try(UserFactory.getByUserId(claims.subject)(realm))
-    } yield action(UserInfo(realm, claims.subject, user.getUsername, None), user, claims))
+    } yield action(UserInfo(realm, claims.subject, user.getUsername, Some(GroupFactory.getDefaultTenant)), user, claims))
```

Comment:
In this particular case, I would prefer to use an option, as it makes it more secure in case of null.